src/main/java/duke/io/DukeParser.java

```diff
@@ -0,0 +1,109 @@
+package duke.io;
+
+import duke.type.DukeCommand;
+import duke.exception.DukeFileSystemException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * A class that handles anything related to the parsing of information for Duke.
+ */
+public class DukeParser {
+    /**
+     * Constructs a new instance of the DukeParser. No arguments are expected.
+     */
+    public DukeParser() {
+
```

Comment:
I think omitting the constructor would be better for readability. Leaving it blank could lead to confusion.